config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-0.yaml

```diff
@@ -0,0 +1,279 @@
+presubmits:
+  kubernetes-sigs/cluster-api:
+  - name: pull-cluster-api-build-release-1-0
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    branches:
+    # The script this job runs is not in all branches.
+    - ^release-1.0$
+    - ^operator-1.0$
```

Comment:
```suggestion
```
You can drop this branch across the file. There is only an `operator-0.4` branch and it looks like the operator work will be moved to a separate repo.